config/webpack.dev.config.js

```diff
@@ -140,10 +141,12 @@ module.exports = merge(commonConfig, {
           {
             loader: 'postcss-loader',
             options: {
-              plugins: () => [
-                PostCssRtlPlugin(),
-                PostCssAutoprefixerPlugin({ grid: true }),
-              ],
+              postcssOptions: {
+                plugins: [
+                  PostCssAutoprefixerPlugin({ grid: true }),
+                  PostCssRTLCSS(),
```

Comment:
[curious] Does the ordering of these 2 plugins matter here? The ordering is reversed from what it was. My guess is "no", but figured it'd be worth asking.